tests/integration/components/course-visualize-instructors-test.js

```diff
@@ -0,0 +1,40 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { component } from 'ilios-common/page-objects/components/course-visualize-instructors';
+
+// @todo flesh this out [ST 2021/02/24]
```

Comment:
I think this might be overkill on the `@todo` given that we'll probably never remember to delete this even if we ever add new tests.

Reply:
removed. :heavy_check_mark: